samples/webview-samples/multi-tab-browser/main.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Listens for the app launching then creates the window
+ *
+ * @see http://developer.chrome.com/apps/app.runtime.html
+ * @see http://developer.chrome.com/apps/app.window.html
+ */
+window.chrome.app.runtime.onLaunched.addListener(function() {
+  runApp();
+});
+
+/**
+ * Listens for the app restarting then re-creates the window.
+ *
+ * @see http://developer.chrome.com/apps/app.runtime.html
+ */
+window.chrome.app.runtime.onRestarted.addListener(function() {
+  runApp();
+});
+
+/**
+ * Creates the window for the application.
+ *
+ * @see http://developer.chrome.com/apps/app.window.html
+ */
+function runApp() {
+  window.chrome.app.window.create(
+    'browser.html',
+    {'id': 'initialBrowserWindowID', 'state': 'maximized'},
```

Comment:
Removed to match the same as that of the browser sample. I left it as is for the new-window-user-agent sample.